### PR TITLE
Add JavaScript for multiple moves

### PR DESCRIPTION
**Pull request by gtvj**
Introduces the JavaScript, HTML and SASS to:

  * Hide destinations where 'no' or 'clear' are selected
  * Show destinations where 'yes' is selected
  * Hide a destination when a user selects 'remove'

### CODE REVIEW

File: app/views/moves/form.html.slim [hunk 1]
```diff
@@ -12,20 +12,22 @@
...
+      .form-group.controls-optional-section
+        = f.radio_button_fieldset :has_destinations,
+          choices: @form.has_destinations_values,
           inline: true
-        = df.text_area :reasons
-        | Remove
-        = df.check_box :_delete
+      #error_move_destinations
+      .optional-section-wrapper
+        = f.fields_for :destinations do |df|
```
Comment (dan-palmer) [inline app/views/moves/form.html.slim]: Should be a `-` instead of an `=` to avoid dumping out the html as text

Comment (gtvj) [inline app/views/moves/form.html.slim]: Have changed the `-` to a `=` now

File: app/models/forms/move.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 module Forms
   class Move < Forms::Base
     REASONS = %w[ sentencing part_heard trial plea_and_directions witness ]
-    HAS_DESTINATIONS = %w[ yes no unknown ]
+    HAS_DESTINATIONS = %w[ yes no clear_selection ]
```
Comment (cesidio) [inline app/models/forms/move.rb]: Please use I18N translations to change the text of a label. This is the value the will go into the database.

Comment (dan-palmer) [inline app/models/forms/move.rb]: @gtvj bump

**Pull request merged**